Formula/trivy.rb

```diff
@@ -0,0 +1,25 @@
+class Trivy < Formula
+  desc "Vulnerbility scanner for container images, file systems, and Git repos"
+  homepage "https://aquasecurity.github.io/trivy/"
+  url "https://github.com/aquasecurity/trivy/archive/v0.24.2.tar.gz"
+  sha256 "ddbffec10e32de2dfbef14169028d60df7504ae255ac674febbafdcf61c7906b"
+  license "Apache-2.0"
+  head "https://github.com/aquasecurity/trivy.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "go" => :build
```

Comment:
Duplicate line

Reply:
😆

Reply:
I think at one moment, I was thinking to use `docker` for `test`, it turns out it is not needed.